app/src/main/java/com/github/openwebnet/repository/impl/EnvironmentRepositoryImpl.java

```diff
@@ -102,4 +106,19 @@ public EnvironmentRepositoryImpl() {
         });
     }
 
+    @Override
+    public Observable<Void> delete(Integer id) {
+        return Observable.create(subscriber -> {
+            try {
```

Comment:
Perhaps we could try deleting items using ```DomoticModel.class``` instead of each individual class. That would do The Right Thing. But I dunno if Realm supports that. What do you think ?